tests/integration/components/bs-form/element/control/switch-test.js

```diff
@@ -0,0 +1,16 @@
+import { module } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { testBS4 } from '../../../../../helpers/bootstrap';
+import hbs from 'htmlbars-inline-precompile';
+import setupNoDeprecations from '../../../../../helpers/setup-no-deprecations';
+
+module('Integration | Component | bs form/element/control/switch', function (hooks) {
+  setupRenderingTest(hooks);
+  setupNoDeprecations(hooks);
+
+  testBS4('it renders', async function (assert) {
+    await render(hbs`<BsForm::Element::Control::Switch />`);
```

Comment:
As above I would prefer to only use public API in tests:

```suggestion
    await render(hbs`
      <BsForm as |form|>
        <form.element @controlType="switch" />
      </BsForm>
    `);
```